packages/hardware-trezor/src/TrezorKeyAgent.ts

```diff
@@ -14,9 +15,9 @@ import {
   errors
 } from '@cardano-sdk/key-management';
 import { txToTrezor } from './transformers/tx';
-import TrezorConnectWeb from '@trezor/connect-web';
 
 const TrezorConnectNode = Trezor.default;
```

Comment:
Do you know why is this weird import style needed? Is it worth leaving a comment?

Reply:
@marcin-mazurek no, i think nobody really knows - @mchappell helped me with this because they had the same weird issue with ledger

Reply:
It should really be equivalent. It indicates that webpack is bundling `cjs` build rather than `esm`. I don't mind merging this, but someone should audit and cleanup lace builds.